tock/tock/views.py

```diff
@@ -14,3 +15,8 @@ def csrf_failure(request, reason=""):
         )
     )
     return render(request, '403.html')
+
+
+def logout(request):
```

Comment:
I decided to add the view to the project, rather than an individual app, because logging out doesn't seem specific to any app in particular.